Robust.Shared/Serialization/Markdown/Validation/FieldNotFoundErrorNode.cs

```diff
@@ -5,7 +5,9 @@ namespace Robust.Shared.Serialization.Markdown.Validation;
 
 public sealed class FieldNotFoundErrorNode : ErrorNode
 {
+    public Type Type;
     public FieldNotFoundErrorNode(ValueDataNode key, Type type) : base(key, $"Field \"{key.Value}\" not found in \"{type}\".", false)
     {
+        Type = type;
```

Comment:
```suggestion
    public Type FieldType { get; }
    public FieldNotFoundErrorNode(ValueDataNode key, Type type) : base(key, $"Field \"{key.Value}\" not found in \"{type}\".", false)
    {
        FieldType = type;
```

1. make it get-only
2. `FieldType` is a more clear name.